examples/pgvector-chunk-embedder/README.md

```diff
@@ -0,0 +1,48 @@
+# Embed And Ingest Data Job Example
+
+The following Versatile Data Kit example allows you to embed document data and metadata (in certain format)
+and ingest it into Postgres instance with pgvector.
+
+# Expected input format
+
+```json
+[
+    {
+        "metadata": {
+            "title": "Page (or chunk) title",
+            "id": "Content page ID",
+            "source": "Source URL",
+            "deleted": <is the content being deleted in the source>
+        },
+        "data": "Content Text"
+    },
+]
+```
+
+# Chunking the data prior to embedding
+
+The example supports configurable chunking - the CHUNK_SIZE and CHUNK_OVERLAP are configured in config.py.
```

Comment:
Can the same chunking logic be applied if we have another vector store (e.g pinecone, FAISS) . In other words would it work?

If it works, then how can be decoupled?

Reply:
How can we vary the chunking? 

Should we consider make it possible to have more structured chunking? For example in HTML maybe you chunk by headers (h1, h2). In Wiki by sections . In Git by functions/ classes .